packages/lexical-editor/src/index.tsx

```diff
@@ -40,6 +37,11 @@ export { CodeHighlightPlugin } from "~/plugins/CodeHighlightPlugin/CodeHighlight
 export { ClickableLinkPlugin } from "~/plugins/ClickableLinkPlugin/ClickableLinkPlugin";
 export { BlurEventPlugin } from "~/plugins/BlurEventPlugin/BlurEventPlugin";
 export { LexicalUpdateStatePlugin } from "~/plugins/LexicalUpdateStatePlugin/LexicalUpdateStatePlugin";
+export { FontColorPlugin } from "~/plugins/FontColorPlugin/FontColorPlugin";
+export { TypographyPlugin } from "~/plugins/TypographyPlugin/TypographyPlugin";
+export { QuotePlugin } from "~/plugins/WebinyQuoteNodePlugin/WebinyQuoteNodePlugin";
+export { WebinyListPlugin } from "~/plugins/WebinyListPLugin/WebinyListPlugin";
```

Comment:
Remove the `Webiny` prefix. Of course it's Webiny, we're in the Webiny repository.